ts_scripts/install_dependencies.py

```diff
@@ -72,22 +72,22 @@ class Linux(Common):
 
     def __init__(self):
         super().__init__()
-        os.system(f"{self.sudo_cmd}apt-get update")
+        if args.reinstall_dependencies == "yes":
+            os.system(f"{self.sudo_cmd}apt-get update")
```

Comment:
Is there any advantage doing this change? It seems like this the update command won't run if reinstall_dependencies is 'no'?

Reply:
correct, I think running an `apt-get update` on a user machine is a tad aggressive so this is deliberate